### PR TITLE
monasca: fix configuration glitches

### PR DESCRIPTION
**Pull request by jgrassler**
This commit fixes the two configuration glitches described in bsc#1077231 by

* Exposing monitor_ceph in the Crowbar UI
* Correcting the type of monasca['agent']['plugins']['libvirt']['ping_check']
  and providing a new default value.

### CODE REVIEW

**Comment by jgrassler** [comment]
Note: I haven't tested this, yet, so please do not merge for now.

**Comment by jgrassler** [comment]
This needed a little more tweaking but now it should work.

File: chef/data_bags/crowbar/template-monasca.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
               "tenant_name"
               ],
             "nova_refresh": 14400,
-            "ping_check": false,
+            "ping_check": "/bin/ip netns exec NAMESPACE /usr/bin/ping",
```
Comment (nicolasbock) [inline chef/data_bags/crowbar/template-monasca.json]: Could you point out to me where we use this attribute?

Comment (jgrassler) [reply chef/data_bags/crowbar/template-monasca.json]: Of course, it's a bit hidden indeed. Here:

https://github.com/crowbar/crowbar-openstack/blob/master/chef/cookbooks/monasca/providers/agent_plugin_libvirt.rb#L29

Basically we dump this data structure straight into the YAML based plugin configuration file.

Comment (nicolasbock) [reply chef/data_bags/crowbar/template-monasca.json]: Wow, that's really hidden :smile:. I did a simple string search in this repository, but that of course didn't find anything useful. Thanks!

Comment (nicolasbock) [reply chef/data_bags/crowbar/template-monasca.json]: But what replaces the `NAMESPACE` string?

Comment (jgrassler) [reply chef/data_bags/crowbar/template-monasca.json]: https://github.com/openstack/monasca-agent/blob/stable/pike/monasca_agent/collector/checks_d/libvirt.py#L624 - that one is even further down the rabbit hole :-)

Comment (jgrassler) [reply chef/data_bags/crowbar/template-monasca.json]: And for completeness' sake, here's the `False` that caused me to wrongly assume a boolean setting, initially: https://github.com/openstack/monasca-agent/blame/stable/pike/docs/Libvirt.md#L62

Comment (nicolasbock) [reply chef/data_bags/crowbar/template-monasca.json]: OMG!

**Comment by jgrassler** [comment]
Note: this needs https://github.com/crowbar/crowbar-openstack/pull/1529 in place to pass CI.

**Pull request merged**